e2e/node/e2e.test.ts

```diff
@@ -251,6 +243,64 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
         "http://www.w3.org/ns/auth/acl#Read",
       ]);
     });
+
+    it("will issue an access request, grant access to a resource, but will not update the ACR if the updateAcr flag is set to false", async () => {
```

Comment:
I'd prefer to avoid spying on calls for the end-to-end tests, and keeping the tests focused on tangible external elements. in this case, checking the value of the ACR after the call shows that it hasn't been updated as we expect, and it doesn't make any assumptions on how that update may have happened.

I'll add an e2e test for when the flag is set to true.

Reply:
Hmm, okay! That's fine then, just as long as we have test coverage asserting that the call is definitely made somewhere.

Reply:
https://github.com/inrupt/solid-client-access-grants-js/pull/432/commits/7ff815ac24a15f7acb69549e3054e3d4e30c37a7